paper-radio-button.html

```diff
@@ -39,6 +39,9 @@
 `--paper-radio-button-unchecked-ink-color` | Selected/focus ripple color when the input is not checked | `--primary-text-color`
 `--paper-radio-button-checked-color` | Radio button color when the input is checked | `--default-primary-color`
 `--paper-radio-button-checked-ink-color` | Selected/focus ripple color when the input is checked | `--default-primary-color`
+`--paper-radio-button-size` | Size of the outer radio button | `16px`
+`--paper-radio-button-inner-size` | Size of the inner radio button | `50%`
```

Comment:
Do you think this should be configurable, or just always 50%? I think that letting people much around with the inner size will make some awful looking buttons, so I'd take that power away from them :)